<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFinanceiroContas</class>
 <widget class="QWidget" name="WidgetFinanceiroContas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1426</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableVencidos">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="TableView" name="tableVencer">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Vencer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vencidos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="3">
        <widget class="ScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>135</width>
            <height>596</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBoxFiltros">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>Filtros</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Status</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="radioButtonTodos">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Todos</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonPendente">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Pendente</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonConferido">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Conferido</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonAgendado">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Agendado</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonRecebido">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Recebido</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonPago">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Pago</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonCancelado">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Cancelado</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBoxLojas">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Loja</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="ItemBox" name="itemBoxLojas">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBoxVencimento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Vencimento</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item row="1" column="1">
                  <widget class="QDateEdit" name="dateEditVencimentoAte">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDateEdit" name="dateEditVencimentoDe">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>De</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Até</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBoxRealizado">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Realizado</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QDateEdit" name="dateEditRealizadoDe">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDateEdit" name="dateEditRealizadoAte">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>De</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Até</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>R$</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxDe">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="prefix">
                    <string>R$ </string>
                   </property>
                   <property name="maximum">
                    <double>99999.990000000005239</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxAte">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="prefix">
                    <string>R$ </string>
                   </property>
                   <property name="maximum">
                    <double>99999.990000000005239</double>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>De</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Até</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="LineEdit" name="lineEditBusca">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelContasReceberBuscar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Buscar:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEditSomaSelecao">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QFrame" name="frame_3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonAbrirDANFE">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Abrir DANFE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdiantarRecebimento">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Antecipar recebimento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonExcluirLancamento">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Excluir lançamento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRemessaItau">
            <property name="text">
             <string>Gerar remessa ITAÚ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonImportarFolhaPag">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Importar folha pag.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInserirLancamento">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Inserir lançamento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInserirTransferencia">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Inserir transferência</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReverterPagamento">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Reverter pagamento</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="TableView" name="table">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>src/scrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableVencidos</tabstop>
  <tabstop>tableVencer</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>radioButtonTodos</tabstop>
  <tabstop>radioButtonPendente</tabstop>
  <tabstop>radioButtonConferido</tabstop>
  <tabstop>radioButtonAgendado</tabstop>
  <tabstop>radioButtonRecebido</tabstop>
  <tabstop>radioButtonPago</tabstop>
  <tabstop>radioButtonCancelado</tabstop>
  <tabstop>groupBoxLojas</tabstop>
  <tabstop>itemBoxLojas</tabstop>
  <tabstop>groupBoxVencimento</tabstop>
  <tabstop>dateEditVencimentoDe</tabstop>
  <tabstop>dateEditVencimentoAte</tabstop>
  <tabstop>groupBoxRealizado</tabstop>
  <tabstop>dateEditRealizadoDe</tabstop>
  <tabstop>dateEditRealizadoAte</tabstop>
  <tabstop>doubleSpinBoxDe</tabstop>
  <tabstop>doubleSpinBoxAte</tabstop>
  <tabstop>pushButtonInserirTransferencia</tabstop>
  <tabstop>table</tabstop>
  <tabstop>lineEditBusca</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
